engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -397,6 +398,7 @@ namespace dmGraphics
         uint32_t        m_Size;
         Type            m_Type;
         bool            m_Normalize;
+        uint64_t        m_NameHash;
```

Comment:
This was added so that we can map a buffer stream to a vertex declaration (in res_mesh) now that a buffer resource can be linked to a HBuffer directly that has been created without DDF data. It's a bit unfortunate that we exposed the VertexElement struct since this might be a breaking change. If we go this route, should we expose the DM_DECLARE_VERTEX_ELEMENT macro and make sure people use that as well?

Reply:
This is fixed since the stream declaration changes